src/main/java/com/amazon/ion/bytecode/bin11/bytearray/ListOpcodeHandlers.kt

```diff
@@ -0,0 +1,197 @@
+        val containerLength = containerSizeUIntValueAndLength.toInt()
+        val prefixLength = containerSizeUIntValueAndLength.shr(Int.SIZE_BITS).toInt()
+        BytecodeEmitter.emitList(destination) {
+            var p = position + prefixLength
+            val end = p + containerLength
+            while (p < end) {
+                val opcode = source[p++].unsignedToInt()
+                p += OpcodeHandlerTable.handler(opcode).convertOpcodeToBytecode(
+                    opcode,
+                    source,
+                    p,
+                    destination,
+                    constantPool,
+                    macroSrc,
+                    macroIndices,
+                    symbolTable,
+                )
+            }
+        }
+        return containerLength + prefixLength
+    }
+}
+
+/**
+ * Writes a delimited list to the bytecode buffer. Handles opcode `0xF0`.
+ */
+internal object DelimitedListOpcodeHandler : OpcodeToBytecodeHandler {
+    @OptIn(ExperimentalStdlibApi::class)
+    override fun convertOpcodeToBytecode(
+        opcode: Int,
+        source: ByteArray,
+        position: Int,
+        destination: BytecodeBuffer,
+        constantPool: AppendableConstantPoolView,
+        macroSrc: IntArray,
+        macroIndices: IntArray,
+        symbolTable: Array<String?>
+    ): Int {
+        assert(opcode == 0xf0) { "Handler cannot compile opcode ${opcode.toHexString()}" }
+
+        var p = position
+        BytecodeEmitter.emitList(destination) {
+            while (true) {
+                val opcode = source[p++].unsignedToInt()
+                if (opcode == OpCode.DELIMITED_CONTAINER_END) {
+                    break
+                }
+                p += OpcodeHandlerTable.handler(opcode).convertOpcodeToBytecode(
+                    opcode,
+                    source,
+                    p,
+                    destination,
+                    constantPool,
+                    macroSrc,
+                    macroIndices,
+                    symbolTable,
+                )
+            }
+        }
+        val bytesRead = p - position
+        return bytesRead
+    }
+}
+
+/**
+ * Writes a tagless-element list to the bytecode buffer. Handles opcode `0x5B`.
+ */
+internal object TaglessElementListOpcodeHandler : OpcodeToBytecodeHandler {
+    @OptIn(ExperimentalStdlibApi::class)
+    override fun convertOpcodeToBytecode(
+        opcode: Int,
+        source: ByteArray,
+        position: Int,
+        destination: BytecodeBuffer,
+        constantPool: AppendableConstantPoolView,
+        macroSrc: IntArray,
+        macroIndices: IntArray,
+        symbolTable: Array<String?>
+    ): Int {
+        assert(opcode == 0x5b) { "Handler cannot compile opcode ${opcode.toHexString()}" }
+
+        var p = position
+        val childOpcode = source[p++].unsignedToInt()
+        val macroAddress = when (childOpcode) {
+            in 0x00..0x47 -> childOpcode
+            in 0x48..0x4f, 0xf4 -> {
```

Comment:
These opcodes do not have the same way of encoding the address. `F4` does have the address as a `FlexUInt`, whereas `48`-`4F` use the system [described here](https://amazon-ion.github.io/ion-docs/books/ion-1-1/binary/e_expressions.html#e-expressions-with-extended-addresses).

```suggestion
            in 0x48..0x4f -> {
                val flexUIntValueAndLength = PrimitiveDecoder.readFlexUIntValueAndLength(source, p)
                val addressLength = flexUIntValueAndLength.shr(Int.SIZE_BITS).toInt()
                p += flexUIntValueAndLength
                val lsb = childOpcode - 0x48
                val msb = flexUIntValueAndLength.toInt() * 8
                msb + lsb + 72
            }
            0xf4 -> {
```